linker/jvm/src/test/scala/org/scalajs/linker/RunTest.scala

```diff
@@ -0,0 +1,66 @@
+/*
+ * Scala.js (https://www.scala-js.org/)
+ *
+ * Copyright EPFL.
+ *
+ * Licensed under Apache License 2.0
+ * (https://www.apache.org/licenses/LICENSE-2.0).
+ *
+ * See the NOTICE file distributed with this work for
+ * additional information regarding copyright ownership.
+ */
+
+package org.scalajs.linker
+
+import scala.concurrent._
+
+import org.junit.{Rule, Test}
+import org.junit.Assert._
+import org.junit.rules.TemporaryFolder
+
+import org.scalajs.ir.Trees._
+
+import org.scalajs.junit.async._
+
+import org.scalajs.jsenv._
+import org.scalajs.jsenv.nodejs.NodeJSEnv
+
+import org.scalajs.linker.interface._
+import org.scalajs.linker.testutils._
+import org.scalajs.linker.testutils.LinkingUtils._
+import org.scalajs.linker.testutils.TestIRBuilder._
+
+class RunTest {
+  import scala.concurrent.ExecutionContext.Implicits.global
+
+  @(Rule @scala.annotation.meta.getter)
+  val tempFolder = new TemporaryFolder()
```

Comment:
Is this something standard that I should know about, or does this deserve a comment? (I feel like it could be the former.)